docs/_sidebar.md

```diff
@@ -1,6 +1,6 @@
 <!-- docs/_sidebar.md -->
 
-- [Reference]()
+- **Reference**
```

Comment:
At the time of creating #22 I didn't know how sidebar rendering works in docsify. This change makes the entry a category header, and the last two items are one indentation level too low, so the result will be a flattened list:
![image](https://github.com/Wyliemaster/gddocs/assets/43796685/348bb97e-ccb6-472a-8077-40550334eed0)
Without `**` the text would be unaligned with Request and Credits due to the arrow being gone.
![image](https://github.com/Wyliemaster/gddocs/assets/43796685/a0046c1b-d972-4c18-91d8-71c9fd8445b3)

For consistency this should just stay as `- [Reference](/)`, with `/` for clarity regarding what the link actually points to. For some reason the last two sidebar sections use this category header formatting, but touching this would be done in a separate PR.